ethereum/contracts/query/QueryResponse.sol

```diff
@@ -0,0 +1,220 @@
+// contracts/query/QueryResponse.sol
+// SPDX-License-Identifier: Apache 2
+
+pragma solidity ^0.8.0;
```

Comment:
Can we use a more modern version of the compiler (e.g. 0.8.19).

Reply:
After an offline discussion, it was agreed to stick with `^0.8.0` since it allows the use of more modern compilers (up to but not including `0.9.0`) but does not place an unnecessary restriction on integrators.